### PR TITLE
MSRV 1.70, remove `once_cell` dependency

### PR DESCRIPTION
**Pull request by robertbastian**
Based on #4971

### CODE REVIEW

File: .cargo/config.toml [hunk 1]
```diff
@@ -6,3 +6,10 @@
 quick = "make quick"
 tidy = "make tidy"
 clippy-all = "clippy --all-features --all-targets -- -D warnings"
+
+[unstable]
+# Use `cargo +nightly update` to update MSRV-aware
+msrv-policy = true
+
+[resolver]
+something-like-precedence = 'something-like-rust-version'
```
Comment (Manishearth) [inline .cargo/config.toml:15]: nit: please link to docs for these fields (I know they're primarily documented in PRs right now)

File: Cargo.toml [hunk 2]
```diff
@@ -268,7 +268,6 @@ indexmap = "1.0.0"
 itertools = "0.10.3"
 ndarray = "0.15.0"
 nom = "7.0.0"
-once_cell = "1.15.0"
```
Comment (Manishearth) [inline Cargo.toml:271]: question: was there no code using this?

**Review by Manishearth — APPROVED** [review]

File: Cargo.toml (hunk 2 above)
Comment (robertbastian) [reply Cargo.toml:271]: There was, and it's updated in this PR. Ignore all the baked data.

Comment (Manishearth) [reply Cargo.toml:271]: ah it's just not showing up on my phone because of the baked data. I'll wait until I have a computer

**Review by Manishearth — APPROVED** [review]

File: Cargo.toml [hunk 3]
```diff
@@ -98,7 +98,7 @@ exclude = [
 
 [workspace.package]
 version = "1.5.0"
-rust-version = "1.67"
+rust-version = "1.70"
```
Comment (sffc) [inline Cargo.toml:101]: Thought: I think our policy allows 1.72? We could update all the way.

**Review by sffc — APPROVED** [review]

File: Cargo.toml (hunk 3 above)
Comment (robertbastian) [reply Cargo.toml:101]: I didn't find anything useful in 1.71-1.73 (which our policy allows in two weeks)

**Pull request merged**